packages/neuron-ui/src/components/NetworkSetting/index.tsx

```diff
@@ -75,20 +83,43 @@ const NetworkSetting = ({ chain = chainState, settings: { networks = [] } }: Sta
     }
   }, [setShowEditorDialog, setNotice, netId])
 
+  const onSwitchNetworkType = useCallback<React.MouseEventHandler<HTMLButtonElement>>(() => {
+    switchCurrentNetworkType()
+  }, [])
```

Comment:
Switching `type` can be transformed into switching `network id` in UI, so logic changes in the backend can be avoided.

Reply:
Fixed by https://github.com/nervosnetwork/neuron/pull/2921/commits/92ae955c62d62bbcc1d4870593bc0f9f57e16e46